torch/_dynamo/variables/builtin.py

```diff
@@ -1271,7 +1273,7 @@ def call_type(self, tx, obj: VariableTracker):
             return VariableBuilder(tx, TypeSource(obj.source))(py_type)
```

Comment:
You can also remove now the TupleVariable if above, as that will be handled by SourcelessBuilder